app/test_main.py

```diff
@@ -1 +1,64 @@
-# write your code here
+from unittest import mock
+import pytest
+
+from app.main import can_access_google_page
+
+
+@pytest.fixture()
+def url() -> None:
```

Comment:
I'm not sure that this function return None?
And I'm not sure that you need a function for this

Reply:
Isn't it better to change the URL in one place rather than in each test?